src/ux/confirmInformation/ConfirmInformation.tsx

```diff
@@ -68,13 +71,15 @@ export function ConfirmInformation ({ displayMode, chainId, address, providerUse
           <Title><Trans>Wallet address</Trans>:</Title>
           {peerWallet && <Title><Trans>Connected wallet</Trans>:</Title>}
           <Title><Trans>Network</Trans>:</Title>
+          {isHardwareWallet && <Title><Trans>Derivation path</Trans>:</Title>}
           {sd && Object.keys(sd.claims).map(key => <Title key={`claim-key-${key}`}>{key}:</Title>)}
           {sd && Object.keys(sd.credentials).map(key => <Title key={`credential-key-${key}`}>{key}:</Title>)}
         </Column>
         <Column>
           <Description>{shortAddress(address)}</Description>
           {peerWallet && <Description>{peerWallet.name}</Description>}
           <Description>{chainId && getChainName(chainId)}</Description>
+          {isHardwareWallet && <Description>{provider.dpath || provider.path}</Description>}
```

Comment:
Ledger returns `provider.dpath` and Trezor returns `provider.path`. I am unsure what dcent returns.

Reply:
https://github.com/rsksmart/rLogin-providers/blob/05980f86e3723e0c3e8d1126e63365ba596fb511/packages/rlogin-dcent-provider/src/DCentProvider.ts#L15 `.path`